controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/DeploymentMetricsMaintainer.java

```diff
@@ -83,6 +88,7 @@ protected void maintain() {
         pool.shutdown();
         try {
             pool.awaitTermination(30, TimeUnit.MINUTES);
+            log.log(Level.WARNING, String.format("Number of application with 0 qps: %d/%d", zeroQps.get(), applicationList.size()));
```

Comment:
Hvert 30 minutt så kan vi sette dette som info. Foren del applikasjoner er jo dette korrekt ikke sant_

Reply:
True, made changes in amended commit.